Extensions/DraggableBehavior/draggableruntimebehavior.ts

```diff
@@ -145,25 +101,188 @@ namespace gdjs {
     }
 
     doStepPostEvents(runtimeScene) {
-      DraggableRuntimeBehavior.leftPressedLastFrame = runtimeScene
+      const mouseDraggableManager = DraggableManager.getMouseManager(
+        runtimeScene
+      );
+      mouseDraggableManager.leftPressedLastFrame = runtimeScene
         .getGame()
         .getInputManager()
         .isMouseButtonPressed(0);
     }
 
     isDragged(runtimeScene): boolean {
-      return this._dragged;
+      return !!this._draggedByDraggableManager;
+    }
+  }
+
+  /**
+   * Handle the dragging
+   */
+  abstract class DraggableManager {
+    protected _draggingSomething = false;
+    protected _xOffset: number = 0;
+    protected _yOffset: number = 0;
+
+    constructor(runtimeScene: gdjs.RuntimeScene) {}
+
+    /**
+     * Get the platforms manager of a scene.
+     */
+    static getMouseManager(
+      runtimeScene: gdjs.RuntimeScene
+    ): MouseDraggableManager {
+      // @ts-ignore
+      if (!runtimeScene.mouseDraggableManager) {
```

Comment:
We could have a script analyzing the codebase to check all usage of `runtimeScene.xxx`. A better way would be in the future to use a storage per runtime scene that is also scoped to each extension.